src/app/shared/dso-page/dso-page-subscription-button/dso-page-subscription-button.component.html

```diff
@@ -0,0 +1,6 @@
+<button *ngIf="isAuthorized$ | async" data-test="subscription-button"
+        (click)="openSubscriptionModal()"
+        [ngbTooltip]="'item.page.subscriptions.tooltip' | translate"
```

Comment:
This button is failing accessibility tests because it has no `title` or `aria-label`. Please add one or both.  They can simply use the same i18n key.